apps/src/templates/curriculumCatalog/expanded_curriculum_catalog_card.module.scss

```diff
@@ -131,6 +135,8 @@ $container-width: 960px;
   white-space: nowrap;
 
   > p {
+    overflow: hidden;
+    white-space: normal;
```

Comment:
What do these do?

Reply:
This helps with the text-wrapping of the subjects list if it's too long. In double checking though, this should only be applied to the subjects section, not all `iconWithDescription` so I'll pull this out! And it looks like `overflow` isn't doing anything so I'll remove that too!

Reply:
I updated the screenshot of the expanded card to show the fix!

Reply:
Ah got it! Thanks for updating 🙌